resources/sass/global/_navbar.scss

```diff
@@ -2,31 +2,64 @@
 .navbar{
     background-color: $primaryColor;
     color: $secondaryColor;
-    position: fixed;
-    top: 0;
-    z-index: 1;
-    width: 100%;
-    &__dropdown{
-        height: 100%;
-        background-color: $primaryColor;
-        margin: 3.3rem -1rem null -1rem;
+    padding: 0.25rem 0.5rem;
+
+    &-wrapper {
+        pointer-events: none;
         position: fixed;
-        width: 100%;
         top: 0;
-        &-item{
+        z-index: 1;
+        height: 100%;
+        width: 100%;
+
+        & * {
+            pointer-events: auto;
+        }
+    }
+
+    &__button {
+        background: none;
+        border: none;
+        color: $white;
+        font-size: 1.65rem;
+
+        &:hover {
+            color: $grey--lighter;
+        }
+    }
+
+    &__dropdown{
+        background-color: $primaryColor;
+        overflow: hidden;
+        height: 0;
+
+        &--show {
+            padding: 0.5rem;
```

Comment:
Please change this to `padding: 0.5rem 0` in order to prevent the nav items from moving to the right/left slightly as the menu opens/closes.